publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/config/RabbitMqPropertiesConfig.java

```diff
@@ -61,6 +61,35 @@ public class RabbitMqPropertiesConfig {
     private static final String GENERATE_SERVER_URI = "generate.server.uri";
     private static final String GENERATE_SERVER_PATH = "generate.server.contextpath";
 
+    private static final String PROPERTY_MP = "mp";
+    private static final String PROPERTY_HOST = "host";
+    private static final String PROPERTY_EXCHANGE_NAME = "exchangeName";
+    private static final String PROPERTY_PORT = "port";
+    private static final String PROPERTY_TLSVER = "tls";
+    private static final String PROPERTY_VIRTUAL_HOST = "virtualHost";
+    private static final String PROPERTY_USERNAME = "username";
+    private static final String PROPERTY_PASSWORD = "password";
+    private static final String PROPERTY_DOMAINID = "domainId";
+    private static final String PROPERTY_CHANNELS_COUNT = "channelsCount";
+    private static final String PROPERTY_CREATE_EXCHANGE_IF_NOTEXISTING = "createExchangeIfNotExisting";
+    private static final String PROPERTY_TCP_TIMEOUT = "tcpTimeOut";
+    private static final String PROPERTY_WAIT_FOR_CONFIRMS_TIMEOUT = "waitForConfirmsTimeOut";
+
+    /***
+     * This method is used to get the protocol specific RabbitMq properties.
+     *
+     * @return specific RabbitMa property
```

Comment:
It should be documented that null is returned when the property is not found.

Reply:
added the proper documentation